python_modules/dagster/dagster/_core/storage/cloud_storage_compute_log_manager.py

```diff
@@ -112,7 +112,10 @@ def _on_capture_complete(self, log_key: Sequence[str]):
         self.upload_to_cloud_storage(log_key, ComputeIOType.STDERR)
 
     def is_capture_complete(self, log_key: Sequence[str]) -> bool:
-        return self.local_manager.is_capture_complete(log_key)
+        if self.local_manager.is_capture_complete(log_key):
```

Comment:
If this is true, we have the files locally? Does that mean that they'd also be available in cloud storage (since that's where they'd have been pulled from?)

Reply:
depends on whether this is the read side or write side.  On the write-side, we check for capture complete, then upload.  On the read-side, we check for the local files, then the remote files, then download from remote to local.